modules/monitoring-about-specifying-limits-and-requests-for-monitoring-components.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="about-specifying-limits-and-requests-for-monitoring-components_{context}"]
+= About specifying limits and requests for monitoring components
+
+Specify limits and requests to manage CPU and memory resource consumption for the containers that run monitoring components. 
+Doing so ensures that the containers in the pods have enough resources to run smoothly.
+
+When you define limits for a container, you specify a cap on a container's resource usage, preventing the container from exceeding the specified values for CPU and memory resources.
+
+When you define requests for a container, you specify that a container should only be scheduled on a node with enough resources to fulfill that request.
+The specified amounts of CPU and memory resources are guaranteed to be available for the container.
+
+You can configure settings for all monitoring components, including the following:
+
+* Alertmanager
+* kube-state-metrics
+* monitoring-plugin
+* node-exporter
+* openshift-state-metrics
+* Prometheus
+* Prometheus Adapter
+* Prometheus Operator and its admission webhook service
+* Telemeter Client
+* Thanos Querier
+* Thanos Ruler
+
```

Comment:
missed Metrics Server
https://github.com/openshift/cluster-monitoring-operator/blob/release-4.15/pkg/manifests/types.go#L132

Reply:
Because this change applies only to 4.15 and because the current PR applies to both 4.14 and 4.15, I am going to leave this info out of the current PR and add it to the enterprise-4.15 branch in a separate PR after the current PR merges.